### ISSUE TITLE
Up and down key doesn't work on the search list when the browser/app size is decreased - reported by @adeel0202

### ISSUE THREAD

**Issue by mvtglobally** [Expensify/App#9735]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/docs/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:
1. Decrease the size of the browser
2. Click on the search icon
3. Press the up/down key multiple times

## Expected Result:
The up and down key should work

## Actual Result:
The up and down key doesn't work

## Workaround:
unknown 

## Platform:

Where is this issue occurring?

- Web
- Desktop App

**Version Number:** 1.1.79-0
**Reproducible in staging?:** Y 
**Reproducible in production?:** Y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** Any additional supporting documentation 
potentially related to https://github.com/Expensify/App/issues/9734 

https://user-images.githubusercontent.com/43995119/177471678-477ac30a-70ee-41d3-a70c-64ffa300f187.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @adeel0202
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1655398921947709

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by thesahindia** [comment]
I have tested this on staging and it is not reproducible. I think it got fixed by https://github.com/Expensify/App/pull/7702.
(Note: The PR is on staging, so it's reproducible on prod)

**Comment by bfitzexpensify** [comment]
Yep, I can reproduce on prod, but it is fine on staging. Closing this out, if we find it is an issue on prod again we can re-open.
